src/ledger/LedgerManagerImpl.cpp

```diff
@@ -504,6 +504,19 @@ LedgerManagerImpl::syncMetrics()
     mApp.syncOwnMetrics();
 }
 
+void
+LedgerManagerImpl::emitAnyStoredSafeMetaAfterLedgerValidation()
+{
+    if (mLastLedgerMeta)
+    {
+        assert(!mApp.getConfig().UNSAFE_EMIT_META);
```

Comment:
`releaseAssert` on both of these `assert`s

Reply:
Will do.

Reply:
Done.